opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
> Interesting, I wonder why?

Did it wrong, probably. It shouldn't be needed.

```
$ cat struct.cpp; ./a.out 
#include <vector>
#include <string>
#include <iostream>

struct foo {
    std::string x;
    std::vector< int > y;
};

int main() {
    std::string x = "foo";
    std::vector< int > y( 10, 4 );

    foo f { x, y };

    std::cout << "x: " << f.x << std::endl;
    std::cout << "y: ";
    for( auto e : f.y )
        std::cout << e << " ";
    std::cout << std::endl;
}

x: foo
y: 4 4 4 4 4 4 4 4 4 4 

```

> In any case adding the constructor is a good enough workaround. I agree we might want to use an rvalue-reference for data though.

I presume you mean `std::move` in the init list? Then yes.

Regardless, if you remove the constructor and brace-initialise this problem goes away.